packages/frontapp/src/app/components/Navbar/index.tsx

```diff
@@ -33,7 +33,7 @@ const NavbarBottomSubsectionWrapper = styled.div`
   position: fixed;
   bottom: 0;
 
-  @media (max-height: 747px) {
+  @media (max-height: 784px) {
```

Comment:
we have `dimensions.ts` file, where we store all the sizes of components and containers, i would recommend moving max-height to dimensions.ts and use it as import

Reply:
Can't find any usage for that dimension, but I will consider that